resources/lib/modules/getSources.py

```diff
@@ -649,6 +649,9 @@ def _get_hosters(self, info, provider):
             for source in sources:
                 source['type'] = 'hoster'
                 source['release_title'] = source.get('release_title', title)
+                release_title = source['release_title'].lower()
+                quality = source_utils.get_quality(str(release_title))
```

Comment:
Also, this logic doesn't exactly work out...

Take for example the release: `The Avengers 2012 BRRip XviD B4ND1T69`

We would presume that this release is at least a 1080p copy, given that it is tagged as "BRRip". Passing this release title to `source_utils.get_quality(...)` returns `"SD"`, since there is no quality to be matched in the release title.

I don't think this is the right way of doing this.